secantTests/SnapshotTests/SecurityWarningSnapshotTests/SecurityWarningSnapshotTests.swift

```diff
@@ -16,7 +16,7 @@ class SecurityWarningSnapshotTests: XCTestCase {
         let store = Store(
             initialState: .initial
         ) {
-            SecurityWarningReducer(zcashNetwork: ZcashNetworkBuilder.network(for: .mainnet))
```

Comment:
No, all tests are mostly network independent and while dependent it's always testnet. This injection here is just because the view needed the network from reducer. Newly the reducer can access it directly from TCA dependency.